apps/sms/index.html

```diff
@@ -63,7 +63,7 @@
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia_sim_picker/style.css">
     -->
     <!-- Shared code -->
-    <script defer src="shared/js/l10n.js"></script>
+    <script type="application/javascript;version=1.8" defer src="shared/js/l20n.js"></script>
```

Comment:
note: you'll need to change this in the split views as well (views/inbox/index.html and views/conversation/index.html)